xtask/codegen/src/html_kinds_src.rs

```diff
@@ -49,6 +49,8 @@ pub const HTML_KINDS_SRC: KindsSrc = KindsSrc {
         "HTML_DOUBLE_TEXT_EXPRESSION",
         "HTML_SINGLE_TEXT_EXPRESSION",
         "HTML_TEXT_EXPRESSION",
+        "SVELTE_SHORTHAND_PROP",
+        "SVELTE_SHORTHAND_SPREAD_PROPS",
```

Comment:
I don't think we added these nodes to the ungram file. We didn't parse them too. Should we remove them or update the PR?

Reply:
These are from a previous iteration, I'll remove them.

Ideally we would parse these so that we can query for them in lint rules, but that's future work.